examples/waitgroup/waitgroup.go

```diff
@@ -0,0 +1,42 @@
+// To wait for multiple goroutines to finish, we can
+// use a *wait group*.
+
+package main
+
+import (
+    "fmt"
+    "sync"
+    "time"
+)
+
+// This is the function we'll run in every goroutine.
+// Note that a WaitGroup must be passed to functions by
+// pointer.
+func worker(id int, wg *sync.WaitGroup) {
+    fmt.Printf("Worker %d starting\n", id)
+
+    // Sleep to simulate an expensive task.
+    time.Sleep(time.Second)
+    fmt.Printf("Worker %d done\n", id)
+
+    // Notify the WaitGroup that we're done.
+    wg.Done()
+}
+
+func main() {
+
+    // This WaitGroup is used to wait for all the
+    // goroutines launched here to finish.
+    var wg sync.WaitGroup
+
+    // Launch several goroutines and increment the WorkGroup
```

Comment:
```suggestion
    // Launch several goroutines and increment the WaitGroup
```

I think? And one more below. Sorry I had seen this earlier but totally spaced on commenting on it.